README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
+# Biography
+
+
+![](https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAqhAAAAJDNlZGE2ZDY3LTZlYTQtNGU1OS1iZTViLThiNjAzOWRlZmI4Yg.jpg)
+
+## Ying Zeng
+
+https://github.com/zyzyxzy
+
+
+### Introduction
+
+Ying Zeng is currently a **MAPSS** student in *psychology* at the University of Chicago. She graduated from **Peking University** in 2017, with dual bachelor degrees in *psychology* and *economics*.
```

Comment:
Obviously, the use of bold and italics is specifically for the purpose of checking boxes on the assignment, but I like that you were consistent in the use of the two - bold for university and italic for degrees.